helpme/views/helpme.py

```diff
@@ -34,20 +35,20 @@ def form_valid(self, form):
         user_agent = self.request.user_agent
 
         if user_agent.is_mobile:
-            device = "Mobile",
+            device = _("Mobile"),
```

Comment:
Since the is a stored value used by code, we should keep it in English and not be localized.